src/ast/passes/codegen_llvm.h

```diff
@@ -327,6 +327,11 @@ class CodegenLLVM : public Visitor {
     return module_->getDataLayout().getTypeAllocSize(s);
   }
 
+  Value *getContext()
+  {
+    return b_.preserveStaticOffset(ctx_);
```

Comment:
Could we wrap `ctx_` in `preserve_static_offsets` when assigning to it? That should lead to less code changes and prevent future regressions by accidentally using `ctx_` instead of `getContext()`.

Reply:
No longer relevant, since I've move things around to make this a bit cleaner and eliminate this method. (I swear I had already responded to this chain, but it must have gotten lost?)